robot_smach_states/src/robot_smach_states/reset.py

```diff
@@ -75,9 +72,7 @@ def __init__(self, robot, timeout=0.0):
         smach.State.__init__(self, outcomes=["done"])
 
     def execute(self, userdata=None):
-        for arm in self.robot.arms.itervalues():
-            arm.reset()
-            arm.send_gripper_goal('close', timeout=self.timeout)
+        self.robot.reset_all_arms(timeout=self.timeout)
```

Comment:
This changes the code, I am not sure if this would change anything, but just to be sure I would prefer the following:

```suggestion
        self.robot.reset_all_arms(gripper_timout=self.timeout)
```

Reply:
fixed